swagger/specification/ucp/resource-manager/UCP/preview/2023-10-01-preview/examples/ResourceGroups_CreateOrUpdate.json

```diff
@@ -17,6 +17,17 @@
         "id": "/planes/radius/local/resourcegroups/rg1",
         "name": "rg1"
       }
+    },
+    "201": {
+      "body": {
+        "location": "global",
+        "id": "/planes/radius/local/resourcegroups/rg1",
+        "name": "rg1"
+      },
+      "headers": {
+        "azure-asyncoperation": "http://example.com/planes/radius/local/providers/System.Resources/locations/global/operations/abcd",
+        "location": "http://example.com/planes/radius/local/providers/System.Resources/locations/global/operations/abcd"
```

Comment:
are these urls right? for Create/Update resourcegroup "rg1" ?

Reply:
These are examples of the async headers returned by Radius for a long-running operation.

I ran [oav](https://github.com/Azure/oav) over the examples, and it wanted me to add the headers.